<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Confirm</class>
 <widget class="QDialog" name="Confirm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnBook">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Book</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSeats">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Instagram Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Select Seat</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Files.qrc">:/new/prefix1/image/cineapp.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Files.qrc"/>
 </resources>
 <connections/>
</ui>
